src/main/java/ProjectLibraryPartOne/Library.java

```diff
@@ -0,0 +1,77 @@
+package ProjectLibraryPartOne;
+
+import java.util.ArrayList;
+
+public class Library {
+
+    ArrayList<Book> listBooks;
+
+    public Library() {
+        this.listBooks = new ArrayList<>();
+    }
+
+    @Override
+    public String toString() {
+        return "Library{" +
+                "listBooks=" + listBooks +
+                '}';
+    }
+
+    void addBook(Book book) {
```

Comment:
methods we write in service class, not in entity/ If you want include thiese methods for library, you can create LibraryService

Reply:
fixed